### DISCUSSION TITLE
Welcome to square-one Discussions!

### DISCUSSION THREAD

**Discussion by nickpelton** [moderntribe/square-one#692] [General]
## 👋 Welcome!
  We’re using Discussions as a place to discuss RFCs in early proposal stages that aren't quite ready for a PR. Use discussions to: 
  * Ask questions about SQ1
  * Propose ideas for SQ1 in the form of an RFC

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

### Templates
* coming soon

**Comment by ChrisMKindred** [comment]
I really like the idea of having a place to post RFC's attached to the project like this but separate from where we are actually doing the work. Using PR's and Issues for RFC's or general questions has always felt strange to me.  Looking forward to using this more.
